src/clients/inst_autosetup_upgrade.rb

```diff
@@ -315,14 +315,28 @@ def main
         Builtins.foreach(@restore) { |res| Pkg.ResolvableInstall(res, :product) }
         Update.SetDesktopPattern if !Update.onlyUpdateInstalled
 
+        # make sure the packages needed for accessing the installation repository
```

Comment:
That I have seen in the "normal" update process and has been missed in AutoYaST. So I have added too.